apps/emqx_authz/src/emqx_authz.appup.src

```diff
@@ -1,8 +1,7 @@
+%% -*- mode: erlang -*-
 %% Unless you know what you are doing, DO NOT edit manually!!
 {VSN,
-  [{"0.1.1",
-    [{load_module,emqx_authn_utils,brutal_purge,soft_purge,[]}]},
-   {<<".*">>,[]}],
-  [{"0.1.1",
-    [{load_module,emqx_authn_utils,brutal_purge,soft_purge,[]}]},
-   {<<".*">>,[]}]}.
+  [{"0.1.0",[{load_module,emqx_authz_utils,brutal_purge,soft_purge,[]}]},
+   {"0.1.1",[{load_module,emqx_authz_utils,brutal_purge,soft_purge,[]}]}],
```

Comment:
0.1.1 is not released yet.

Reply:
Please help to revert the change

Reply:
Wonder why the version jump was not caught by:
https://github.com/emqx/emqx/blob/466fd52168f78c967150f9e4d004995daf63c978/scripts/apps-version-check.sh#L44

Reply:
The problem is, it has been bumped before 5.0.0 release. This is the version from 5.0.0:

```
{application, emqx_authz, [
    {description, "An OTP application"},
    {vsn, "0.1.1"},
    {registered, []},
    {mod, {emqx_authz_app, []}},
    {applications, [
        kernel,
        stdlib,
        crypto,
        emqx_connector
    ]},
    {env, []},
    {modules, []},

    {licenses, ["Apache 2.0"]},
    {links, []}
]}.
```

So in 5.0.1 the version should be bumped, and the appup should take care of 0.1.1